sdk/dotnet/examples/CSharpScript/main.csx

```diff
@@ -0,0 +1,14 @@
+#r "../PulumiAzure/bin/Debug/netstandard2.1/Pulumi.dll"
+#r "../PulumiAzure/bin/Debug/netstandard2.1/PulumiAzure.dll"
+
+using Pulumi.Azure.Core;
+using Pulumi.Azure.Storage;
+
+var resourceGroup = new ResourceGroup("rg");
+
+var storageAccount = new Account("sa", new AccountArgs
+{
+    ResourceGroupName = resourceGroup.Name,
+    AccountReplicationType = "LRS",
+    AccountTier = "Standard",
+});
```

Comment:
how does this work given that we haven't called Deployment.RunAsync @mikhailshilkov ?

Reply:
C# script doesn't work currently. I think we can ignore it for now.

Reply:
Gotcha.  Want to either remove this, or comment it?  thanks!

Reply:
I'll remove it with other examples.